config.py

```diff
@@ -482,30 +482,49 @@
 # XXX not needed if bids events are present
 # min_event_duration = 0.002
 
-#  `event_id`` : dict
+#  ``event_id`` : dict
 #    Dictionary that maps events (trigger/marker values)
-#    to conditions.
+#    to conditions. The ``/``` character can be used to group conditions.
```

Comment:
```suggestion
#    to conditions. The ``/`` character can be used to group conditions.
```

Reply:
AGAIN one of those!!! Really need to get this sorted out… Thanks for catching this!